### DISCUSSION TITLE
Microsoft Visual Studio Community 2022  - Failed to start host Copilot

### DISCUSSION THREAD

**Discussion by rogeriostranieri** [community/community#19190] [Copilot Conversations]
Microsoft Visual Studio Community 2022 (64-bit)
Version 17.2.4
and installed GitHub.Copilot.Vsix.1.30.0.1, still the problem, Failed to start host

`GitHub Copilot 1.30.0.1 (v1.30.0.1@c4d49dd57)
GitHub Copilot Agent v1.30.0
[Error] An error occurred for 'LSP': StreamJsonRpc.RemoteInvocationException: Failed to start host 'node.x86$C94B8CFE-E3FD-4BAF-A941-2866DBB566FE'. Host exited with exit code '1'. (ErrorKind: Error)
   em StreamJsonRpc.JsonRpc.<InvokeCoreAsync>d__143`1.MoveNext()
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
   em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   em Microsoft.ServiceHub.Client.HubClient.RemoteServiceBrokerWrapper.<>c__DisplayClass9_0.<<RequestServiceChannelAsync>b__0>d.MoveNext()
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
   em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   em Microsoft.ServiceHub.Client.HubClient.RemoteServiceBrokerWrapper.<>c__DisplayClass9_0.<<RequestServiceChannelAsync>b__0>d.MoveNext()
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
   em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   em Microsoft.ServiceHub.Utility.Shared.ServiceHubRetry.<ExecuteAsync>d__0`1.MoveNext()
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
   em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   em Microsoft.ServiceHub.Utility.Shared.ServiceHubRetry.<ExecuteAsync>d__0`1.MoveNext()
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
   em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   em Microsoft.ServiceHub.Client.HubClient.RemoteServiceBrokerWrapper.<RequestServiceChannelAsync>d__9.MoveNext()
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
   em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   em Microsoft.ServiceHub.Client.HubClient.RemoteServiceBrokerWrapper.<RequestServiceChannelAsync>d__9.MoveNext()
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
   em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   em Microsoft.ServiceHub.Framework.RemoteServiceBroker.<GetPipeAsync>d__32.MoveNext()
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
   em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   em Microsoft.ServiceHub.Client.HubClient.<GetPipeAsync>d__38.MoveNext()
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
   em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   em GitHub.Copilot.CopilotLanguageClient.<ActivateAsync>d__40.MoveNext()
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
   em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   em Microsoft.VisualStudio.LanguageServer.Client.RemoteLanguageClientInstance.<InitializeAsync>d__114.MoveNext()
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
   em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   em Microsoft.VisualStudio.Threading.TplExtensions.<WithTimeout>d__5.MoveNext()
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
   em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   em Microsoft.VisualStudio.LanguageServer.Client.RemoteLanguageClientInstance.<TrackTaskAsync>d__115.MoveNext()
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
   em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   em Microsoft.VisualStudio.LanguageServer.Client.RemoteLanguageClientInstance.<ActivateLanguageClientAsync>d__124.MoveNext()
RPC server exception:
Microsoft.ServiceHub.Utility.ServiceHubException: Failed to start host 'node.x86$C94B8CFE-E3FD-4BAF-A941-2866DBB566FE'. Host exited with exit code '1'. (ErrorKind: Error)
      em Microsoft.ServiceHub.Controller.LocationService.<StartServiceAsync>d__60.MoveNext()
   --- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
      em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
      em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
      em Microsoft.ServiceHub.Controller.LocationService.<GetHostedServiceLocationAsync>d__76.MoveNext()
   --- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
      em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
      em Microsoft.ServiceHub.Controller.LocationService.<LocateAsync>d__44.MoveNext()
   --- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
      em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
      em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
      em Microsoft.ServiceHub.Controller.HubController.<LocateAsync>d__8.MoveNext()
   --- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
      em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
      em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
      em Microsoft.ServiceHub.Controller.HubController.<RequestServiceChannelAsync>d__13.MoveNext()
   --- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
      em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
      em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
      em Microsoft.ServiceHub.Controller.ClientConnection.<RequestServiceChannelAsync>d__27.MoveNext()
`

Is there any solution for Microsoft Visual Studio Community 2022 ?

**Comment by thedatakidx** [comment]
I've just restarted the IDE multiple times and it fixed the issue..

Hope this works for you 🙂
